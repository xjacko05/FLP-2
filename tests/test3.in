A B
A C
A D
A G
B C
B E
B H
C F
C I
D E
D F
D G
E F
E H
F I
G H
G I
H I
