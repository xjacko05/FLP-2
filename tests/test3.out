A-B B-C C-F F-D D-E E-H H-I I-G G-A 
A-B B-C C-F F-E E-H H-I I-G G-D D-A 
A-B B-C C-F F-I I-G G-H H-E E-D D-A 
A-B B-C C-F F-I I-H H-E E-D D-G G-A 
A-B B-C C-I I-G G-H H-E E-F F-D D-A 
A-B B-C C-I I-H H-E E-F F-D D-G G-A 
A-B B-C C-I I-F F-D D-E E-H H-G G-A 
A-B B-C C-I I-F F-E E-H H-G G-D D-A 
A-B B-E E-D D-G G-H H-I I-F F-C C-A 
A-B B-E E-D D-F F-C C-I I-H H-G G-A 
A-B B-E E-H H-G G-D D-F F-I I-C C-A 
A-B B-E E-H H-G G-I I-C C-F F-D D-A 
A-B B-E E-H H-I I-C C-F F-D D-G G-A 
A-B B-E E-H H-I I-G G-D D-F F-C C-A 
A-B B-E E-F F-C C-I I-H H-G G-D D-A 
A-B B-E E-F F-D D-G G-H H-I I-C C-A 
A-B B-H H-G G-D D-E E-F F-I I-C C-A 
A-B B-H H-G G-I I-C C-F F-E E-D D-A 
A-B B-H H-E E-D D-G G-I I-F F-C C-A 
A-B B-H H-E E-D D-F F-C C-I I-G G-A 
A-B B-H H-E E-F F-C C-I I-G G-D D-A 
A-B B-H H-E E-F F-D D-G G-I I-C C-A 
A-B B-H H-I I-C C-F F-E E-D D-G G-A 
A-B B-H H-I I-G G-D D-E E-F F-C C-A 
A-C C-B B-E E-D D-F F-I I-H H-G G-A 
A-C C-B B-E E-H H-G G-I I-F F-D D-A 
A-C C-B B-E E-H H-I I-F F-D D-G G-A 
A-C C-B B-E E-F F-I I-H H-G G-D D-A 
A-C C-B B-H H-G G-I I-F F-E E-D D-A 
A-C C-B B-H H-E E-D D-F F-I I-G G-A 
A-C C-B B-H H-E E-F F-I I-G G-D D-A 
A-C C-B B-H H-I I-F F-E E-D D-G G-A 
A-C C-F F-D D-E E-B B-H H-I I-G G-A 
A-C C-F F-E E-B B-H H-I I-G G-D D-A 
A-C C-F F-I I-G G-H H-B B-E E-D D-A 
A-C C-F F-I I-H H-B B-E E-D D-G G-A 
A-C C-I I-G G-H H-B B-E E-F F-D D-A 
A-C C-I I-H H-B B-E E-F F-D D-G G-A 
A-C C-I I-F F-D D-E E-B B-H H-G G-A 
A-C C-I I-F F-E E-B B-H H-G G-D D-A 
A-D D-E E-B B-C C-F F-I I-H H-G G-A 
A-D D-E E-H H-B B-C C-F F-I I-G G-A 
A-D D-E E-F F-C C-B B-H H-I I-G G-A 
A-D D-E E-F F-I I-C C-B B-H H-G G-A 
A-D D-F F-C C-B B-E E-H H-I I-G G-A 
A-D D-F F-E E-B B-C C-I I-H H-G G-A 
A-D D-F F-E E-H H-B B-C C-I I-G G-A 
A-D D-F F-I I-C C-B B-E E-H H-G G-A 
